jams/sonify.py

```diff
@@ -0,0 +1,99 @@
+#!/usr/bin/env python
+# CREATED:2015-12-12 18:20:37 by Brian McFee <brian.mcfee@nyu.edu>
+r'''
+Sonification
+============
+
+.. autosummary::
+    :toctree: generated/
+
+    sonify
+'''
+
+import six
+import numpy as np
+import mir_eval.sonify
+from .eval import validate_annotation
+from .exceptions import NamespaceError
+
+__all__ = ['sonify']
+
+
+def clicks(annotation, sr=22050, length=None, **kwargs):
+    '''Sonify clicks timings
+
+    '''
+
+    interval, _ = annotation.data.to_interval_values()
+
+    return mir_eval.sonify.clicks(interval[:, 0], fs=sr, length=length, **kwargs)
+
+
+def chord(annotation, sr=22050, length=None, **kwargs):
+    '''Sonify chords'''
+
+    intervals, chords = annotation.data.to_interval_values()
+
+    return mir_eval.sonify.chords(chords, intervals, fs=sr, length=length, **kwargs)
+
+
+def pitch_midi(annotation, sr=22050, length=None, **kwargs):
+    '''Sonify midi pitches'''
+
+    intervals, notes = annotation.data.to_interval_values()
+
+    freqs = 440.0 * (2.0 ** ((np.arange(128) - 69.0)/12.0))
+
+    gram = np.zeros((len(freqs), len(notes)))
+
+    for t, n in enumerate(notes):
+        gram[n, t] = 1.0
+
+    return mir_eval.sonify.time_frequency(gram, freqs, intervals[:, 0],
```

Comment:
note start times will be correct but durations will be set arbitrarily based on start time of next note